src/main.py

```diff
@@ -0,0 +1,6 @@
+def hello(name):
```

Comment:
use my style:
https://github.com/akuleshov7/h-yapf

Reply:
If only you could give a guide how to add it to my workflow :)